source/unified-test-format/unified-test-format.md

```diff
@@ -1307,6 +1315,8 @@ The structure of this object is as follows:
 - `readConcern`: Optional object. See [commonOptions_readConcern](#commonOptions_readConcern).
 - `readPreference`: Optional object. See [commonOptions_readPreference](#commonOptions_readPreference).
 - `writeConcern`: Optional object. See [commonOptions_writeConcern](#commonOptions_writeConcern).
+- `encryptedFields`: Optional object. See
+    [Client Side Encryption: `encryptedFields`](../client-side-encryption/client-side-encryption.md#encryptedFields)
```

Comment:
`collectionOrDatabaseOptions` is used to get a collection or database object. They are not expected to be arguments to the [create](https://www.mongodb.com/docs/manual/reference/command/create/) command sent to the server. Suggest moving this option to `createOptions`.